### ISSUE TITLE
Android 11 Smart Home Shortcuts

### ISSUE THREAD

**Issue by Freymi** [Waboodoo/HTTP-Shortcuts#196]
It would be nice if it was possible to execute Requests with the new Android 11 Smart Home Shortcuts:
https://developer.android.com/guide/topics/ui/device-control

**Comment by candiesdoodle** [comment]
This will be awesome.
Not only can this be used to trigger custom http requests..it can also be used to display, say, sensor values like temperature or humidity after reading it from a http request

**Comment by Waboodoo** [comment]
I've started experimenting with this a bit, but I don't yet have a clear vision on how this would best be integrated. I'll likely go with a very basic implementation in an upcoming release, and then gradually improve it with future releases.

**Comment by wrenix** [comment]
Is there a basic integration of "Device Control" in the current release? (The Issue is really old)
It would be nice, to have that feature to trigger thinks without complete unlock the screen.

I just has an old Version build by F-Droid #446.

PS: Thank you for this App it is really cool and  nice.

**Comment by Waboodoo** [comment]
@wrenix No, there has been no progress on this topic. I still want to implement it, at least a basic version of it, but so far I haven't gotten around to it.
I'll try to look into it once I have the https://github.com/Waboodoo/HTTP-Shortcuts/issues/446 resolved.

**Comment by Waboodoo** [comment]
There is now a very basic implementation of Device Control support, as of version 3.23.0 which is now released.
I'm open to feedback and might enhance the feature in the future.

**Comment by wrenix** [comment]
Wow, found it and i will test it the next days. Thank you. for your work 🎁
